src/dmd/backend/ty.d

```diff
@@ -336,7 +336,114 @@ uint tyrelax(tym_t ty) { return _tyrelax[tybasic(ty)]; }
 extern __gshared ubyte[TYMAX] tyequiv;
 
 /* Give an ascii string for a type      */
-extern (C) { extern __gshared const(char)*[TYMAX] tystring; }
+extern (C) __gshared const(char)*[TYMAX] tystring =
```

Comment:
does this still need to be `extern (C)`?

Reply:
I removed the last commit as the tests were failing. Will re-instate later.